app/experimenter/base/__init__.py

```diff
@@ -0,0 +1,25 @@
+import json
+import os
+
+from django.conf import settings
+
+APP_VERSION = None
+
+
+def app_version():
```

Comment:
I had this code living in `context_processors.py` but refactored out into its own function. Seemed easier to test that way and possibly reusable if we need this info anywhere else.